### DISCUSSION TITLE
请问 SchemaField 解析 schema 时，如果 component 没匹配到为什么不报错了？

### DISCUSSION THREAD

**Discussion by liuweiGL** [alibaba/formily#1279] [Q&A 2.x]
今天开始从 1.0 迁移到 2.0 时发现  `SchemaField.XXX` 组件中的 `x-component` 属性没匹配到组件时不会报错了。感觉还是 1.0 那样在控制台提示一下比较好，现在不方便调试。

**Comment by janryWang** [comment]
这个需要加一下，不过得处理__DEV__模式提示，非__DEV__模式不提示的问题，这块还需要涉及构建流程的修改
